.github/ISSUE_TEMPLATE/update_docs.yml

```
@@ -0,0 +1,20 @@
+name: Update docs ✍️
+description: >-
+  Find a mistake in our documentation, or have a suggestion to improve them? Let us know here.
+title: '[Docs]: '
```

Maybe this is less confusing? (thinking of addon docs)
```suggestion
title: '[Documentation]: '
```